lib/jekyll-github-metadata/owner.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module Jekyll
+  module GitHubMetadata
+    class Owner
+      extend Forwardable
+
+      # Defines an instance method that delegates to a hash's key
+      #
+      # hash   - a symbol representing the instance method to delegate to. The
+      #          instance method should return a hash or respond to #[]
+      # key    - the key to call within the hash
+      # method - (optional) the instance method the key should be aliased to.
+      #          If not specified, defaults to the hash key
+      #
+      # Returns a symbol representing the instance method
+      def self.def_hash_delegators(hash, *methods)
+        content_methods.concat(methods)
+        methods.each do |method|
+          define_method(method) do
+            send(hash)[method.to_s]
+          end
+        end
+      end
+
+      def self.content_methods
+        @content_methods ||= []
+      end
+
+      # List of whitelisted keys.
+      def_hash_delegators :owner_info,
+                          :avatar_url,
+                          :bio,
+                          :blog,
+                          :collaborators,
+                          :company,
+                          :created_at,
+                          :description,
+                          :email,
+                          :followers,
+                          :following,
+                          :has_organization_projects,
+                          :has_repository_projects,
+                          :hireable,
+                          :html_url,
+                          :id,
+                          :is_verified,
+                          :location,
+                          :login,
+                          :name,
+                          :node_id,
+                          :public_gists,
+                          :public_gists,
+                          :public_repos,
+                          :public_repos,
+                          :type,
+                          :updated_at
```

Comment:
@parkr  if you don't mind, *may I collapse this list..* to multiple attributes per line?

Reply:
@ashmaroli I'd prefer each attribute to be on their own lines. The reason is that diffs are *much* clearer when we're working with whole lines. This is also an allow list, so for auditing purposes, it makes it a little easier to look at. 😄